physutils/physio.py

```diff
@@ -542,3 +544,55 @@ def neurokit2phys(
             metadata = dict(peaks=peaks)
 
         return cls(data, fs=fs, metadata=metadata, **kwargs)
+
+
+class MRIConfig:
```

Comment:
Maybe I missed it, but is there any test to cover that class ?

Reply:
No, it is not used yet, because it would need detection of the trigger channel in order to initialize it. I can open an issue defining the next steps for it, and I'll implement it if there is time after the workflow